src/docs/010_manual/60_further_reading.adoc

```diff
@@ -6,102 +6,100 @@ ifndef::imagesdir[:imagesdir: ../images]
+** https://www.writethedocs.org/guide/docs-as-code/[write the docs: docs-as-code]
+==== 
 
 === Static Site Generator
+.Static site Generator
+[%collapsible]
+==== 
 
-* https://jbake.org[jBake] is the underlying static site generator for the xref:../015_tasks/03_task_generateSite.adoc[`generateSite`]-Task
-* https://www.docsy.dev/[docsy] is the standard theme we use
-* https://getbootstrap.com/docs/4.6/getting-started/introduction/[twitter bootstrap] is the CSS framework
+* The underlying static site generator for the xref:../015_tasks/03_task_generateSite.adoc[`generateSite`]-Task: https://jbake.org[jBake]
+* Our standard theme: https://www.docsy.dev/[docsy] 
+* Our CSS framework: https://getbootstrap.com/docs/4.6/getting-started/introduction/[twitter bootstrap] 
 
-== Books
-
-NOTE: links to Amazon are affiliate links
-
-=== English Books
-
-* https://www.amazon.de/dp/B0784ZJWSR[DOCS-LIKE-CODE] by Anne Gentle
-* https://amzn.to/2sn0kyx[Modern technical writing:
-An Introduction to Software Documentation] by Andrew Etter
-* https://leanpub.com/arc42byexample[arc42 by Example] by Gernot Starke, Stefan Zörner, Michael Simons, Ralf D. Müller
-* https://leanpub.com/arc42inpractice[Communicating Software Architectures with arc42] by Gernot Starke und Peter Hruschka
-* https://leanpub.com/visualising-software-architecture[Visualise, document and explore your software architecture] by Simon Brown
+==== 
 
-=== German Books
-
-* https://amzn.to/2smM5Kp[arc42 in Aktion:
-Praktische Tipps zur Architekturdokumentation] von Gernot Starke und Peter Hruschka
-* https://amzn.to/2ru9Yvr[Softwarearchitekturen dokumentieren und kommunizieren:
-Entwürfe, Entscheidungen und Lösungen nachvollziehbar und wirkungsvoll festhalten] von Stefan Zörner
-
-== Past and upcoming Talks
-
-=== Dokumentation am (Riesen-)Beispiel – arc42, AsciiDoc und Co. in Aktion
+== Books
 
-https://twitter.com/gernotstarke[Gernot Starke] and https://twitter.com/ralfdmueller[Ralf D.
-Müller]
+NOTE: These are Amazon affiliate links.
 
-Anhand eines großen Systems zeigen Gernot und Ralf, wie Sie mit ziemlich wenig Aufwand angemessene und vernünftige Dokumentation für unterschiedliche Stakeholder produzieren – sodass Entwicklungsteams dabei auch noch Spaß haben.
 
-Unser Rezept:
-AsciiDoc mit arc42 mischen, Automatisierung mit Gradle und Maven hinzufügen und mit Diagramm- oder Modellierungstools Ihrer Wahl kombinieren.
-Schon sind schicke HTML- und reviewfähige PDF-Dokumente fertig.
-Auf Wunsch gibts DOCX und Confluence als Zugabe.
+[width="80%",cols="2,^2,^10",options="header"]
+|=========================================================
+|Title |Author |Language
 
-Wir zeigen, wie Sie Doku genau wie Quellcode verwalten können, stakeholderspezifische Dokumente erzeugen und Diagramme automatisiert integrieren können.
-Einige Teile dieser Doku können Sie sogar automatisiert testen.
+|https://www.amazon.de/dp/B0784ZJWSR[DOCS-LIKE-CODE] | Anne Gentle | English 
```

Comment:
This link does not work, check with Ralf as it's an affiliate link. Also, change the title to Docs Like Code (not DOCS-LIKE-CODE).

Reply:
Just replace it with the non-affiliated link:
https://www.amazon.com/dp/1387081322